### DISCUSSION TITLE
Transform commands on lamellar morphology?

### DISCUSSION THREAD

**Discussion by mustafasubasic** [neperfepx/neper#657] [Q&A] [answered]
Dear all

I am trying to perform some -transform commands on a simple lamellar morphology (see image below) but without success. Is it possible to do transformation commands such as cut(cylinder(), hspace()) on a lamellar morphology?

Thank you!

Best regards
Mustafa Subasic

![pic](https://user-images.githubusercontent.com/126886091/224268863-6570be5d-017a-4963-be64-6322b42aadce.png)

Neper inputs:

neper -T -n "from_morpho::id==1?1:1000" -domain "cube(2,1,1)" -morpho "lamellar(w=1:1,v=(1,0,0),pos=start)" -o d
neper -V d.tess -print pic

**Comment by rquey** [comment, marked as answer]
No, it is not possible. The `cut` transformation makes use of phantom seeds, are there are no seeds to phantom in the case of a lamellar morphology...

**Reply by mustafasubasic** [reply]
Thank you very much for the answer!
